src/index.ts

```diff
@@ -3351,10 +3368,53 @@ export interface PreferencePurposesTabExperienceLayoutConfig {
 }
 
 /** Preferences - Subscriptions Experience Layout Config */
+
+export enum UnsubscribeAllImpact {
+  Universal = 'universal',
+  SingleExperience = 'singleExperience',
+}
+
+export enum SubscriptionItemType {
+  Unknown = 0,
```

Comment:
why are we all the sudden using numeric enums?  String enums are easier to work with (and don't have the "our BE cannot send `0`" problem)

Reply:
good point, I had just copied this over from Saravanan's subscription work. Updated in [97bb1ad](https://github.com/ketch-sdk/ketch-types/pull/151/commits/97bb1ad4c8412776783bf6dfb32de21067c97742) to use string enums